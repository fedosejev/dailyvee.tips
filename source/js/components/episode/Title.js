import React from 'react';

const Title = ({ videoUrl, number, name }) => (
  <a href={videoUrl} target="_blank">
    <h2>
      <span className="episodeNumber">#{number} </span>
      {name}
    </h2>
  </a>
);

export default Title;
