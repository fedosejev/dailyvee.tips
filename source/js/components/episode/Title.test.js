import React from 'react';
import Title from './Title';
import renderer from 'react-test-renderer';

describe('Title', () => {
  test('SHOULD render correctly', () => {
    const tree = renderer.create(
      <Title />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
