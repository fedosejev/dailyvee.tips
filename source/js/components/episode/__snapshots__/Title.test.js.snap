// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title SHOULD render correctly 1`] = `
<a
  href={undefined}
  target="_blank"
>
  <h2>
    <span
      className="episodeNumber"
    >
      #
       
    </span>
  </h2>
</a>
`;
